scripts/generate-test-contracts.sh

```diff
@@ -35,7 +35,7 @@ sed -i.bak "s/ColonyDataTypes/UpdatedColonyDataTypes/g" ./contracts/IUpdatedColo
 sed -i.bak "s/contract IUpdatedColony is UpdatedColonyDataTypes, IRecovery {/contract IUpdatedColony is UpdatedColonyDataTypes, IRecovery {function isUpdated() public pure returns(bool);/g" ./contracts/IUpdatedColony.sol
 # Modify UpdatedReputationMiningCycle contract
 sed -i.bak "s/contract ReputationMiningCycle/contract UpdatedReputationMiningCycle/g" ./contracts/UpdatedReputationMiningCycle.sol
-sed -i.bak "s/WAD;/WAD;function isUpdated() public pure returns(bool) {return true;}/g" ./contracts/UpdatedReputationMiningCycle.sol
+sed -i.bak "s|// 24 hours|// 24 hours\nfunction isUpdated() public pure returns(bool) {return true;}|g" ./contracts/UpdatedReputationMiningCycle.sol
```

Comment:
Remind me what's happening here?

Reply:
We're generating an 'upgraded' `reputationMiningCycle` to use in the upgrade tests. Because the constant that was at the start of this contract has been moved to `reputationMiningCycleCommon`, we need to change the regex used here.

Reply:
Mkay